### PR TITLE
[AAP-8147] Application Tokens List

### PR DESCRIPTION
**Pull request by prat98**
This pr is related to Jira issue [AAP-8147](https://issues.redhat.com/browse/AAP-8147)

### CODE REVIEW

File: frontend/awx/administration/applications/ApplicationPage/ApplicationPageTokens.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        titleHelp={t('List of access tokens associated with this application.')}
+        titleDocLink={`${getDocsBaseUrl(config)}/html/userguide/applications_auth.html`}
+      />
+      <PageTable<Token>
+        id="awx-applications-token-table"
+        toolbarFilters={toolbarFilters}
+        toolbarActions={toolbarActions}
+        tableColumns={tableColumns}
+        rowActions={rowActions}
+        errorStateTitle={t('Error loading applications')}
```
Comment (AlexSCorey) [inline frontend/awx/administration/applications/ApplicationPage/ApplicationPageTokens.tsx:78]:
```suggestion
        errorStateTitle={t('Error loading tokens')}
```

File: frontend/common/columns.tsx [hunk 2]
```diff
@@ -271,3 +286,37 @@ export function useTypeColumn<T extends object>(options: {
   );
   return column;
 }
+export function useScopeColumn<T extends { scope?: string }>(options?: {
```
Comment (AlexSCorey) [inline frontend/common/columns.tsx:289]: I believe this column, and the `useExpires` column only exist in the tokens list...they are one-off columns.  In the interest of WET (write everything twice) before abstracting it out I believe we should move these to live inside the useTokensColumns file.

File: frontend/awx/administration/applications/hooks/useTokensFilters.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  const toolbarFilters = useMemo<IToolbarFilter[]>(() => [nameToolbarFilter], [nameToolbarFilter]);
+  return toolbarFilters;
+}
+
+function useTokenUserToolbarFilter() {
+  const { t } = useTranslation();
+  return useMemo<IToolbarFilter>(
+    () => ({
+      key: 'name',
+      label: t('name'),
```
Comment (AlexSCorey) [inline frontend/awx/administration/applications/hooks/useTokensFilters.tsx:16]:
```suggestion
      label: t('Name'),
```

**Review by AlexSCorey — APPROVED** [review]
Looks really good.  Just a few minor changes.  Let me know if you have questions.

File: frontend/common/columns.tsx (hunk 2 above)
Comment (prat98) [reply frontend/common/columns.tsx:289]: I contemplated putting them in the useTokensColumns file but I saw that this page will need it too https://awx-devel.aap-ci-k3s-01.gcp.testing.ansible.com/#/users/3/tokens

**Review by vishsanghishetty — APPROVED** [review]
🚀 lgtm

**Pull request merged**